src/hypofuzz/cov.py

```diff
@@ -34,6 +36,11 @@ def make(fname: str, start: int, end: int) -> "Arc":
             _ARC_CACHE.setdefault(fname, {}).setdefault(start, {})[end] = self
             return self
 
+    def __str__(self) -> str:
+        return f"{self.fname}:{self.start_line}:{self.end_line}"
```

Comment:
```suggestion
        return f"{self.fname}:{self.start_line}::{self.end_line}"
```
The conventional reading would be `fname:lineno:colno`, so I think we want `::` to distinguish this.  Maybe in future we'll get column number too from branch observations?  

(Avoiding redundancy for branchless code would be nice too; iirc there were some comments on Hypothesis about this.)

Reply:
I'd like to compress the arcs for sure, I think we will have to ast analyze though. will probably start by relying on coverage.py for that